### PR TITLE
Feature/71 add a guided tutorial how to add a new pilot

### PR DESCRIPTION
**Pull request by BonnetGa**

### CODE REVIEW

File: docs/tutorials/pilot_tutorial.rst [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+
+5. Implement your pilot
+
+
+Implement your pilot
+--------------------
+
+1. Define the provider_name by adding a new enum value to the ProviderName.py enum. And set it as your provider_name.
+
+2. Define the supported_languages. If necessary add a new enum value to the AssemblerLanguage.py enum or use the already existing one.
```
Comment (st161630) [inline docs/tutorials/pilot_tutorial.rst:23]: or use the already existing ones.

File: docs/tutorials/pilot_tutorial_demo.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
+How to create a new Qunicorn-Pilot
+==================================
+
+1. Create a new python file in the `qunicorn_core.core.pilotmanager` directory. The qunicorn_pilot.py
+
+2. For the creation you can also just copy the base_pilot.py file and rename it. Keep in mind that the supported_languages and provider_name is set in the implement your pilot chapter.
+    .. image:: ../resources/images/tutorial_demo/create_pilot.png
+
+3. After at least creating the Pilot class inside extending from the base_pilot add your pilot to the "PILOTS" list in the pilotmanager.py file.
```
Comment (st161630) [inline docs/tutorials/pilot_tutorial_demo.rst:9]: "After creating a Pilot class extending from the base_pilot in the file, add your..." Possibly a nicer formulation?

File: docs/tutorials/pilot_tutorial_demo.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+1. Define the qunicorn provider by adding a new enum value to the ProviderName.py enum. And set it as your provider_name.
+    .. image:: ../resources/images/tutorial_demo/provider.png
+        :width: 90%
+
+2. Define the supported_languages. Add a new qunicorn enum value to the AssemblerLanguage.py enum.
+    .. image:: ../resources/images/tutorial_demo/assembler.png
+        :width: 90%
+
+3. Implement all me
+thods that raise a "NotImplementedError" in the base_pilot.py file.
```
Comment (st161630) [inline docs/tutorials/pilot_tutorial_demo.rst:30]: layout error

File: docs/tutorials/pilot_tutorial_demo.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+4. Check out the comments in the base_pilot.py file for more information about the methods and also find some examples in the aws_pilot or ibm_pilot file.
+
+5. Add a json file to add a default qunicorn device and add the json in the pilot_resources folder. It is important that the name of the file is ProviderName_standard_devices, in our case qunicorn_standard_devices, so it will be used automatically.
+    .. image:: ../resources/images/tutorial_demo/all_devices_json.png
+
+
+Test if the current pilot already works
+---------------------------------------
+1. Therefor we need to restart the database: "flask recreate-and-load-db" (If this throws an error delete your db-instance first and check in the readme if the command changed)
```
Comment (st161630) [inline docs/tutorials/pilot_tutorial_demo.rst:49]: Therefore

File: docs/tutorials/pilot_tutorial_demo.rst [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        :width: 80%
+
+2. And also use them in the test_utils. Create there both static variables: DEPLOYMENT_QUNICORN_CIRCUITS_JSON="deployment_request_dto_with_qunicorn_circuit_test_data.json" and JOB_JSON_QUNICORN="job_request_dto_test_data_QUNICORN.json" And add them to the get_test_jobs/deployments:
+    .. image:: ../resources/images/tutorial_demo/use-test-data.png
+        :width: 90%
+
+3. After that a new test file can be created in automated_tests: test_qunicorn_job_execution.py to use the test method and test the pilot end to end:
+    .. image:: ../resources/images/tutorial_demo/tests.png
+
+4. Do not forget to write comments and documentation to your new implemented code.
```
Comment (st161630) [inline docs/tutorials/pilot_tutorial_demo.rst:102]: newly implemented code

File: docs/tutorials/pilot_tutorial.rst (hunk 1 above)
Comment (st162107) [reply docs/tutorials/pilot_tutorial.rst:23]: "an already existing one"

File: docs/tutorials/pilot_tutorial_demo.rst (hunk 2 above)
Comment (st162107) [reply docs/tutorials/pilot_tutorial_demo.rst:9]: Better, but logically not quite correct - changed the old formulation

**Review by st162107 — APPROVED** [review]

**Pull request merged**